project/src/main/ui/settings/settings-menu-bottom.gd

```diff
@@ -63,6 +66,8 @@ func _refresh_quit_type() -> void:
 		$HBoxContainer/VBoxContainer1/Holder1.visible = false
 		$HBoxContainer/VBoxContainer2/Spacer.visible = false
 		$HBoxContainer/VBoxContainer3/Spacer.visible = false
+	
+	_tab_input_enabler.refresh_focus_neighbours_for_current_tab()
```

Comment:
Worth commenting this line, as it's a bit strange

Reply:
Added comment, ` 	# refresh focus neighbors for the TabContainer and newly visible buttons`